src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeUp.groovy

```diff
@@ -14,6 +14,8 @@ import java.time.Instant
 class ComposeUp extends DefaultTask {
 
     private Map<String, ServiceInfo> servicesInfos = new HashMap<>()
+    private List<String> composeFiles
+    private boolean filesSpecified

```

Comment:
I think that `extension.useComposeFiles` would be fine, so we don't need this field (if I didn't overlook something).